test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -1339,6 +1342,242 @@ defmodule Trento.SapSystemTest do
   end
 
   describe "deregistration" do
+    test "should deregister a single DB instance if no SR enabled" do
```

Comment:
I'm totally lost with the order of things.
Why this goes first in the `deregistration` describe?
Let's have some logic or time based order please. I see total random additions